pkgs/applications/window-managers/cagebreak/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv, fetchFromGitHub
+, meson, ninja, pkg-config, wayland, scdoc, makeWrapper
+, wlroots, wayland-protocols, pixman, libxkbcommon
+, cairo , pango, fontconfig, pandoc, udev
+, xwayland ? null
```

Comment:
this does nothing since it's already defined in `all-packages.nix`

Reply:
Fixed. I added an option to compile without xwayland.